src/App.js

```diff
@@ -1,19 +1,62 @@
 import React from 'react';
+
+import TodoList from './component/TodoList/TodoList';
+import NewTodo from './component/NewTodo/NewTodo';
+
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const usersMap = users
+  .reduce((acum, user) => ({ ...acum, [user.id]: user }), {});
+
+function getTodosWithUsers(todosArray) {
+  return todosArray.map(todo => ({
+    ...todo,
+    user: usersMap[todo.userId],
+  }));
+}
+
+const preparedTodos = getTodosWithUsers(todos, users);
+
+class App extends React.Component {
+  state = {
+    listTodos: [...preparedTodos],
```

Comment:
```suggestion
    todos: [...preparedTodos],
```
а массив с файла назови `todosApi`